client/src/components/CommentApp/main.scss

```
@@ -72,8 +72,9 @@ $box-padding: 10px;
     }
   }
 
-  > :last-child &__notice {
-    bottom: -$box-padding;
+  & > &__notice-placeholder:last-child &__notice,
+  &__replies:last-child > :last-child &__notice
+  {
```

Opening curly braces should be on the final line of a selector:

```suggestion
  & > &__notice-placeholder:last-child &__notice,
  &__replies:last-child > :last-child &__notice {
```

---

Thanks for looking over it. I have effected this change.